### ISSUE TITLE
Internationalisation / Select Time Zone

### ISSUE THREAD

**Issue by wortkrieg** [wortkrieg/streamdeck-onairclock#2]
Would be useful if multiple instances of the button on one page can display different time zones, like in a news room.

**Comment by wortkrieg** [comment]
May require a label text to know which timezone it is.

**Comment by pmad99** [comment]
On the note of internationalization, some countries look at the date as MM-DD, others look at it as DD-MM. Can we get an option to swap Month and Day?

**Comment by pmad99** [comment]
> On the note of internationalization, some countries look at the date as MM-DD, others look at it as DD-MM. Can we get an option to swap Month and Day?

You know.. i might be losing my mind today... I just noticed the clock is a 24 hour clock instead of 12 hours AM/PM, and i looked to see if this was an option i could change that i might have missed... Low and behold, the Date Format is there! 

so about that 12/24 hour option? haha... wait.. am i just not seeing that option somewhere?

**Comment by wortkrieg** [comment]
You're right, I haven't built a 12h am/pm time format into the plugin. Will do this when I find some spare time :)

**Comment by wortkrieg** [comment]
Finally built the 12/24 hour option, but will not build the timezones anytime soon
